packages/web/src/scss/components/Button/index.html

```diff
@@ -19,46 +19,72 @@ <h2 class="docs-Heading">Variants</h2>
     <h2 class="docs-Heading">Sizes</h2>
 
     <div class="mb-400">
-      <button type="button" class="Button Button--primary">Medium button</button>
+      <button type="button" class="Button Button--primary Button--medium">Primary medium button</button>
+      <button type="button" class="Button Button--secondary Button--medium">Secondary medium button</button>
+      <button type="button" class="Button Button--tertiary Button--medium">Tertiary medium button</button>
+      <button type="button" class="Button Button--danger Button--medium">Danger medium button</button>
     </div>
     <div class="docs-Box my-600">
-      <button type="button" class="Button Button--inverted">Medium inverted button</button>
+      <button type="button" class="Button Button--inverted Button--medium">Inverted medium button</button>
     </div>
     <div class="mb-400">
-      <button type="button" class="Button Button--primary Button--large">Large button</button>
+      <button type="button" class="Button Button--primary Button--large">Primary large button</button>
+      <button type="button" class="Button Button--secondary Button--large">Secondary large button</button>
+      <button type="button" class="Button Button--tertiary Button--large">Tertiary large button</button>
+      <button type="button" class="Button Button--danger Button--large">Danger large button</button>
     </div>
     <div class="docs-Box my-600">
-      <button type="button" class="Button Button--inverted Button--large">Large inverted button</button>
+      <button type="button" class="Button Button--inverted Button--large">Inverted large button</button>
+    </div>
+
+    <div class="mb-400">
+      <button type="button" class="Button Button--primary Button--medium Button--square">🚀</button>
+      <button type="button" class="Button Button--secondary Button--medium Button--square">🚀</button>
+      <button type="button" class="Button Button--tertiary Button--medium Button--square">🚀</button>
+      <button type="button" class="Button Button--danger Button--medium Button--square">🚀</button>
+    </div>
+    <div class="docs-Box my-600">
+      <button type="button" class="Button Button--inverted Button--medium Button--square">🚀</button>
+    </div>
+
+    <div class="mb-400">
+      <button type="button" class="Button Button--primary Button--large Button--square">🚀</button>
+      <button type="button" class="Button Button--secondary Button--large Button--square">🚀</button>
+      <button type="button" class="Button Button--tertiary Button--large Button--square">🚀</button>
+      <button type="button" class="Button Button--danger Button--large Button--square">🚀</button>
+    </div>
+    <div class="docs-Box my-600">
+      <button type="button" class="Button Button--inverted Button--large Button--square">🚀</button>
     </div>
 
   </section>
   <section class="docs-Section">
 
     <h2 class="docs-Heading">Square Button</h2>
```

Comment:
I believe these two sections can be merged together. Emoji examples can be deleted since we have icons at our disposal.

And when we are on it, I'd change the `profile` icon to something more common, e.g. `hamburger`, `more`, or `plus`.

![obrazek](https://user-images.githubusercontent.com/5614085/191253496-e769a8ef-d339-4b30-a3ac-f822300fa424.png)